packages/arb-token-bridge-ui/src/util/SentryUtils.ts

```diff
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/react'
+
+export function captureSentryErrorWithExtraData({
+  error,
+  originFunction,
+  erc20ParentAddress,
+  additionalData
+}: {
+  error: unknown
+  originFunction: string
+  erc20ParentAddress?: string
+  additionalData?: Record<string, string>[]
```

Comment:
For type-safety (And to ensure we don't create new tag-names on the fly), we should have the keys belonging to an enum like `type TagName = 'origin function' | 'type' | 'token' ....`

and hence, the `additionalData` type can change to `additionalData: Record<TagName, string>[]`

Reply:
I think it should be flexible and not to restrict what they are